pipelines/DECam/SingleFrame.yaml

```diff
@@ -0,0 +1,23 @@
+description: >-
+  Single-frame pipeline for the case in which
+  no templates exist, specialized for DECam
+instrument: lsst.obs.decam.DarkEnergyCamera
+imports:
+  - location: $AP_PIPE_DIR/pipelines/DECam/ProcessCcd.yaml
+    include: 
+      - processCcd
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/SingleFrame.yaml
+    include:
+      - getRegionTimeFromVisit
+      - mpSkyEphemerisQuery
+      - ssSingleFrameAssociation
+      - initialPviCore
+subsets:
+  processCcd:
+    subset:
+      - isr
+      - calibrateImage
+      - ssSingleFrameAssociation
+      - initialPviCore
```

Comment:
I'd leave out `initialPviCore`, for consistency with the definition in `_ingredients` (though I agree we probably will want to add it everywhere in the future).